admin_guide/managing_networking.adoc

```diff
@@ -85,6 +85,40 @@ can now access any pods and services in the cluster and vice versa.
 Alternatively, instead of specifying specific project names, you can use the
 `--selector=<project_selector>` option.
 
+[[admin-guide-disabling-hostname-collision]]
+== Disabling Host Name Collision Detection For Ingress Objects
+
+In {product-title}, host name collision prevention is enabled by default. This
+means that if an ingress object, for example, a route, is set to a host name, it
```

Comment:
With routes we forbid the host update.  There is no way to disable it.  This just adds the security to ingress that routes already have (it is necessary for a multi-tenant, self-service, environment... but other users don't need this)